frontend/src/presentation/pages/QuestionsPage.tsx

```diff
@@ -177,7 +187,12 @@ const QuestionsPage: React.FC = () => {
                 footer={null}
                 width={1200}
             >
-                <NewQuestionForm />
+                <NewQuestionForm
+                    onSubmit={() => {
+                        handleCloseModal();
+                        fetchQuestions();
```

Comment:
No need to fetch all questions when submitted.

Reply:
Ill fix it in a later PR